submission/programs/adsl.R

```diff
@@ -276,11 +276,11 @@ adsl06 <- adsl05 %>%
     new_var = DURDIS,
     start_date = DISONSDT,
     end_date = VISIT1DT,
-    out_unit = "months",
+    out_unit = "days",
     add_one = TRUE
   ) %>%
   mutate(
-    DURDIS = round(DURDIS, digits = 1)
+    DURDIS = round_sas(DURDIS/(365.25/12), digits = 1)
```

Comment:
Can we add a code comment to explain why we're taking this approach rather than just deriving DURDIS in months directly in the call to `derive_vars_duration()`, for the sake of any external reviewers?

Reply:
Thanks for the review, @DeclanHodges ! 
Comments are added

Reply:
No problem @kaz462, thanks for updating!

Reply:
Thanks, both!